pending.json

```diff
@@ -419,6 +419,24 @@
         "description": "Probably related to SMS based on the name?\nContains no services and I've never seen it run.",
         "removal": "caution"
     },
+    {
+        "id": "com.oplus.aod",
+        "label": "Aod",
+        "description": "Multiple sources say that AOD doesnt work without tons of extra services, but on some devices it works without extra services",
```

Comment:
I didn't understand the description. What does _extra services_ mean?

Reply:
By *extra services* I mean other system apps